crates/miden-objects/src/errors.rs

```diff
@@ -388,6 +399,8 @@ pub enum AssetVaultError {
 
 #[derive(Debug, Error)]
 pub enum NoteError {
+    #[error("note tag tag length {0} exceeds the maximum of {max}", max = NoteTag::MAX_LOCAL_TAG_LENGTH)]
+    NoteTagTagLengthTooLarge(u8),
```

Comment:
```suggestion
    #[error("note tag length {0} exceeds the maximum of {max}", max = NoteTag::MAX_LOCAL_TAG_LENGTH)]
    NoteTagLengthTooLarge(u8),
```

Nit: Wouldn't that say the same?